src/main/java/Todo.java

```diff
@@ -0,0 +1,13 @@
+public class Todo extends Task {
```

Comment:
Same as the question in `Duke.java`. Perhaps we can put classes in a package `exception` etc.